.gitignore

```diff
@@ -1,6 +1,5 @@
 .bundle
 bin/configlet
 bin/configlet.exe
-Gemfile.lock
```

Comment:
I don't know why the `Gemfile.lock` wasn't checked in, but it normally should be, so I've added it.